### ISSUE TITLE
[16.0] project_list module is not useful anymore

### ISSUE THREAD

**Issue by ByteMeAsap** [OCA/project#1373]
## Module
project_list

## Describe the bug
This module is not needed anymore as the list view is added in default project module in Odoo.
Check https://github.com/odoo/odoo/blob/d1be9e4a4ec2cf6eaafe982b5b0036afa5421207/addons/project/views/project_views.xml#L984 and https://github.com/odoo/odoo/blob/d1be9e4a4ec2cf6eaafe982b5b0036afa5421207/addons/project/views/project_views.xml#L1004

## To Reproduce
**Affected versions**:
16.0

**Expected behavior**
The module can be removed from the 16.0 branch

**Comment by pedrobaeza** [comment]
Yes, indeed. The usual way is to empty the module and change the README to comment it, for not having broken installations.
